examples/simple-nft-module/src/tests.rs

```diff
@@ -0,0 +1,27 @@
+use sov_modules_api::default_context::DefaultContext;
```

Comment:
Could we please have tests to be not inside `src`, but in the module root?

Reply:
We had a long discussion offline. In my opinion this should be solved as a separate issue. @citizen-stig  Let me know if you are ok with that and I will create the issue.

Reply:
https://github.com/Sovereign-Labs/sovereign-sdk/issues/993